apis/nucleus/src/utils/handlers/data-property-handlers.js

```diff
@@ -0,0 +1,83 @@
+import { getFieldById } from './handlers-helpers';
+
+class DataPropertyHandler {
```

Comment:
The class source is in sense-client [data-hypercube-handler.ts](https://github.com/qlik-trial/sense-client/blob/main/web/assets/objects/utils/data-properties/data-property-handler.ts)